neonvm/controllers/virtualmachinemigration_controller.go

```diff
@@ -666,11 +668,14 @@ func (r *VirtualMachineMigrationReconciler) doFinalizerOperationsForVirtualMachi
 // Note that the Pods will be also watched in order to ensure its
 // desirable state on the cluster
 func (r *VirtualMachineMigrationReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	cntrlName := "virtualmachinemigration"
+	reconciler := WithMetrics(r, r.Metrics, cntrlName)
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&vmv1.VirtualMachineMigration{}).
 		Owns(&corev1.Pod{}).
 		WithOptions(controller.Options{MaxConcurrentReconciles: 8}).
-		Complete(r)
+		Named(cntrlName).
```

Comment:
not a blocker: what's the impact of `Named` here?

Reply:
Sets the name of the controller. Its used in metrics particularly.

https://github.com/kubernetes-sigs/controller-runtime/blob/b88ed7a3602b85b6cfd0acfe9c25033b978bdb83/pkg/builder/controller.go#L210-L214

Reply:
Ah fair enough - so this is more a continuation of #739?

Reply:
Right, I want to expose the metrics in the same format at kubebuilder does.